src/notebooks/finance/interventions/kamishima.ipynb

```diff
@@ -125,8 +131,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "PR = PrejudiceRemover(eta=5.0, sensitive_attr=\"sex\", class_attr=\"salary\")\n",
-    "PR.fit(train_sds)"
+    "PR = joblib.load(artifacts_dir / \"models\" / \"finance\" / \"kamishima.pkl\")"
```

Comment:
I tried this the other day, it saves something to a temporary location on your machine, so it works locally but fails once you commit. I think it's better to just create it and call `.fit` in the notebook. Can you take a look at whether it's possible to fix a random seed maybe?